cmd/hivechain/generate.go

```diff
@@ -200,6 +202,14 @@ func (g *generator) setDifficulty(i int, gen *core.BlockGen) {
 	}
 }
 
+func (g *generator) setParentBeaconRoot(i int, gen *core.BlockGen) {
+	if g.genesis.Config.IsCancun(gen.Number(), gen.Timestamp()) {
+		var h common.Hash
+		rand.Read(h[:])
```

Comment:
The chain won't be deterministic anymore with this. We could change to math/rand and static seed or just use a hash function or something.